src/EnergyPlus/SystemAvailabilityManager.cc

```diff
@@ -2808,13 +2857,9 @@ namespace SystemAvailabilityManager {
 				}
 
 			} else if ( SELECT_CASE_var == AdaptiveTemperatureGradient ) {
-				NumPreDays = OptStartSysAvailMgrData( SysAvailNum ).NumPreDays;
-				if ( OptStartSysAvailMgrData( SysAvailNum ).CtrlType == ControlZone ) {
-					if ( ! allocated( AdaTempGradTrdHeat ) ) {
```

Comment:
I guess you found that these `allocated` checks are just not needed?  It's always nice to remove checks like this when we know for sure they aren't needed since the compiler surely can't optimize these out.